src/help/help.go

```diff
@@ -64,6 +68,103 @@ func help(topic string) string {
 	if f, present := m[topic]; present {
 		return helpFromBuildRule(f.FuncDef)
 	}
+
+	// Check plugins
+	if pluginHelp := helpForPlugin(topic); pluginHelp != "" {
+		return pluginHelp
+	}
+
+	return ""
+}
+
+func helpForPlugin(topic string) string {
+	config, err := core.ReadDefaultConfigFiles(nil)
+	if err != nil {
+		panic("Failed to read config")
+	}
+	if _, ok := config.Plugin[topic]; ok {
+		message := fmt.Sprintf("${BOLD_BLUE}%v${RESET} is a plugin defined in the .plzconfig file.", topic)
+		if val, ok := config.Plugin[topic].ExtraValues["subrepo"]; ok {
+			subrepo := val[0]
+			buildLabel := core.BuildLabel{PackageName: "", Name: subrepo, Subrepo: subrepo}
+			state := newState()
+
+			// Parse the subrepo (Run reads the plugin config into config)
+			plz.Run([]core.BuildLabel{buildLabel}, nil, state, config, state.TargetArch)
+			downloadBuildLabel := core.BuildLabel{PackageName: "", Name: "_" + subrepo + "#download", Subrepo: ""}
```

Comment:
Is this just to try and figure out where it's downloaded from? Might be a little grungy as it depends on the internal structure of the build target. It also seems to assume the target is in the root of the repo. You could use a label though. If the subrepo rule labels itself somehow e.g. `labels = ["plugin_repo:https://github.com/please-build/python-rules"]`, you can fetch that off the target directly without having to delve into the details of the rule.

Reply:
So it looks like it's not super straightforward to get the URL from the remote_file rule because there can be multiple URLs attached to the rule. I think one day/in another PR it might be nice for the remote_file to label itself with what URL it ultimately went with.